### ISSUE TITLE
support for laravel 6

### ISSUE THREAD

**Issue by priosshrsth** [mlanin/laravel-api-exceptions#9]
When will it supoort laravel 6?

**Comment by naillizard** [comment]
added support here, hopefully will be merged soon #10, you can however configure your composer.json to use my fork for the time being.

```json
#composer.json
...
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/naillizard/laravel-api-exceptions"
        }
    ],
"require": {
        "php": "^7.2",
        "fideloper/proxy": "^4.0",
        "laravel/framework": "^6.2",
        "laravel/tinker": "^1.0",
        "lanin/laravel-api-exceptions": "dev-laravel-6-support"
    },
...
```

**Comment by naillizard** [comment]
@priosshrsth I think this has been merged, check v2.0, now on L7 support.

**Comment by naillizard** [comment]
@mlanin I think this one can be closed, also created PR for Laravel 7 support, kindly accept.
